### PR TITLE
Integrationtest

### PR DESCRIPTION
**Pull request by SamYuan1990**
add Integration test.

- as sub module issue for az, use git clone to checkout fabric-sample
- download fabric cli, docker etc...
- running byfn
- go build and test with byfn

### CODE REVIEW

File: cicd.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+#git submodule update --init --recursive
```
Comment (guoger) [inline cicd.sh:2]: remove unused line (or add explanation why this should be kept)

File: cicd.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh -o ./scripts/bootstrap.sh
+chmod +x ./scripts/bootstrap.sh
+./scripts/bootstrap.sh
+cd ./first-network
+sed -i 's/^askProceed/#askProceed/g' byfn.sh
+./byfn.sh up
+cd ../..
+cp -r ./fabric-samples/first-network/crypto-config .
+go build
+./stupid config.json 10
```
Comment (guoger) [inline cicd.sh:14]: a simpler version could be
```
#!/usr/bin/env bash
set -e

DIR=$PWD

curl -vsS https://raw.githubusercontent.com/hyperledger/fabric/master/scripts/bootstrap.sh | bash
cd ./fabric-samples/first-network
echo y | ./byfn.sh up
cp -r crypto-config "$DIR" && cd "$DIR"
go build
./stupid config.json 10
```

File: azure-pipelines.yml [hunk 3]
```diff
@@ -9,22 +9,40 @@ trigger:
...
+    steps:
+     # - script: |
+     #     git config --global --add filter.lfs.required true
+     #     git config --global --add filter.lfs.smudge "git-lfs smudge -- %f"
+     #     git config --global --add filter.lfs.process "git-lfs filter-process"
+     #     git config --global --add filter.lfs.clean "git-lfs clean -- %f"
+     #   displayName: Configure LFS for use with submodules
+     # - checkout: self
+     #   lfs: true
+     #   submodules: true
```
[the remark this answers is not in the document]
Comment (SamYuan1990) [reply azure-pipelines.yml]: done

File: cicd.sh (hunk 1 above)
Comment (SamYuan1990) [reply cicd.sh]: removed

File: cicd.sh (hunk 2 above)
Comment (SamYuan1990) [reply cicd.sh]: thanks

**Comment by SamYuan1990** [comment]
Well, you are able to take a look...
as UT and build are duplicate for go get...

**Pull request merged**